fe/src/app/jobs/[jobId]/page.tsx

```diff
@@ -0,0 +1,102 @@
+import Carousel from "@/components/Carousel/Carousel";
+import Timer from "@/components/Timer/Timer";
+import { Button, Chip, Typography } from "@/lib/materialTailwindExports";
+import { AdvertType } from "@/types/AdvertType";
+import { getToken } from "@/utils/getToken";
+import { BASE_API_URL, SERVICE_ADVERTRS_ROUTE } from "@/utils/urls";
+import { FaCheckCircle } from "react-icons/fa";
+import { FaCircleXmark } from "react-icons/fa6";
+
+const AnswerIcon = (isPositive: boolean) => isPositive ? <FaCheckCircle className="text-green-500 text-2xl" /> : <FaCircleXmark className="text-red-500 text-2xl" />;
+
+async function getAdvert(jobId: string) {
+  const token = await getToken();
+  const response = await fetch(`${BASE_API_URL}${SERVICE_ADVERTRS_ROUTE}/${jobId}`, {
+    headers: { Authorization: `Bearer ${token}` }
+  });
+  if (!response.ok) {
+    return;
+  }
+
+  return response.json();
+}
+
+export default async function AdvertInfoPage({ params: { jobId } }: { params: { jobId: string }}) {
+  const advert: AdvertType = await getAdvert(jobId);
+
+  if (!advert) {
+    return "Not found";
+  }
+
+  return (
+    <div className="flex flex-col gap-3 max-w-3xl mx-auto my-5 py-10 px-5">
+      <Typography variant="h3">
+        {advert.title}
+      </Typography>
+      <div className="flex justify-between">
+        <div className="flex gap-5">
+          <Chip size="lg" variant="gradient" className="rounded-full chip-opacity" value={advert.brand} />
+          <Chip size="lg" variant="gradient" className="rounded-full chip-opacity" value={advert.model} />
+          <Chip size="lg" variant="gradient" className="rounded-full chip-opacity" value={advert.manufactureYear} />
+          <Chip size="lg" variant="gradient" className="rounded-full chip-opacity" value={advert.problemType} />
+        </div>
+        <Chip
```

Comment:
the chip should have a border even when not selected, because it was hard to see and I missed to fill it naturally
<img width="595" alt="image" src="https://github.com/MatasAdmoneo/CarFans/assets/99315244/e74f450b-78c3-4610-a6e6-2a745bb6db47">

Reply:
Added border.